about.html

```diff
@@ -0,0 +1,108 @@
+          </ul>
+          <button class="campaign" type="button">CC Campaign</button>
+        </nav>
+      </header>
+    
+      <section id="About-Introduction">
+        <p class="greetings">"Hello! Developers!"</p>
+        <h1 class="event-title">INTERNATIONAL JAVASCRIPT SUMMIT 2022</h1>
+        <div class="separator"></div>
+        <p class="event-description">The International JavaScript Summit brings together each 2 years the community of experts and academics.
+             From Angular and React to WebAssembly; from Progressive Web Apps to JAMstack, international experts share insights on the present and future
+              JavaScript.  Join us to broaden your knowledge and network with other JS enthusiasts.              
+        </p>
+
+        <div class="contact-us">
+            <p>Please contact us per Email for any further questions about IJS Summit 2022! <br />
+                <a class="email" href="mailto:ijssummit2022@ijshaiti.org">ijssummit2022@ijshaiti.org <i class="fas fa-chevron-right"></i></a>
+            </p>            
+        </div>
+      </section>
+    </main>
+
+
+   <section id="ccgs2015-logo">
+      <h2 class="ccgs2015-logo-title">International JavaScript Summit 2022 Logo</h2>
+      <div class="separator"></div>      
+      <p class="ccgs2015-logo-description">
+          The logo off IJS Summit 2022 was decided through the logo competition from 8, June to 7, July.
+          After Selection Committee's judging process, the logo from Shiller Emmanuel Desrosiers won the prize.
+      </p>      
+      <img class="img-logo" src="images/LogoHemer.png" alt="CCGS 2015 Logo">      
+   </section>
+
+  <section id="Past-ccgs">
+    <h2 class="Past-ccgs-title">See the past IJS Summits</h2>
+    <div class="separator"></div>
+    <p class="Past-ccgs-description">Take a look at the last two IJS Summits which took place in Buenos Aires 
+        and in Warsaw. 
+    </p>
+    <div class="past-summit">
+      <div class="ccgs2013">
+        <h2>2020</h2>
+        <p>IJS Summit 2020 in Buenos Aires</p>
+      </div>
+      <div class="ccgs2011">
+        <h2>2018</h2>
+        <p>IJS Summit 2018 in Warsaw</p>
+      </div>
+    </div>
+  </section>   
+
+  <section id="Partner">
+    <h2 class="Partner-title">Partner</h2>
+    <div class="separator"></div>
+    <ul class="Partner-list">
+      <li><img class="Partner-list-item" src="images/partners/mozilla.png" alt="mozilla"></li>
+      <li><img class="Partner-list-item" src="images/partners/google.png" alt="google"></li>
+      <li><img class="Partner-list-item" src="images/partners/Naver.png" alt="Naver"></li>
+      <li><img class="Partner-list-item" src="images/partners/kakao.png" alt="Kakao"></li>
+      <li><img class="Partner-list-item" src="images/partners/airbnb.png" alt="Airbnb"></li>
+      <li><img class="Partner-list-item" src="images/partners/infopromologo.webp" alt="infopromo"></li>
+    </ul>
+</section>
+
+  <footer id="footer">
```

Comment:
- [ ] **Required** In the desktop version, the footer background should be dark in `about page`. Kindly fix it.
- Your website
![dvsv](https://user-images.githubusercontent.com/26767383/152563916-6a0200e8-cc90-4ca2-8c58-f05dda529e8c.PNG)
- Original website
![sdcs](https://user-images.githubusercontent.com/26767383/152563985-61684f05-ff8e-44a0-becb-f643bb9326c4.PNG)

Reply:
- [ ] The black footer background should only apply on the about page. The home page footer background should remain white. Kindly apply the same for the mobile view.

Reply:
- [ ] The black footer background should only apply on the about page. The home page footer background should remain white. Kindly apply the same for the mobile view.